include/tables/scene_table.h

```diff
@@ -111,12 +111,12 @@
 /* 0x63 */ DEFINE_SCENE(spot20_scene, g_pn_42, SCENE_LON_LON_RANCH, SDC_LON_LON_RANCH, 0, 0)
 /* 0x64 */ DEFINE_SCENE(ganon_tou_scene, g_pn_43, SCENE_OUTSIDE_GANONS_CASTLE, SDC_OUTSIDE_GANONS_CASTLE, 0, 0)
 // Debug-only scenes
-/* 0x65 */ DEFINE_SCENE(test01_scene, none, SCENE_TEST01, SDC_CALM_WATER, 0, 0)
-/* 0x66 */ DEFINE_SCENE(besitu_scene, none, SCENE_BESITU, SDC_BESITU, 0, 0)
-/* 0x67 */ DEFINE_SCENE(depth_test_scene, none, SCENE_DEPTH_TEST, SDC_DEFAULT, 0, 0)
-/* 0x68 */ DEFINE_SCENE(syotes_scene, none, SCENE_SYOTES, SDC_DEFAULT, 0, 0)
-/* 0x69 */ DEFINE_SCENE(syotes2_scene, none, SCENE_SYOTES2, SDC_DEFAULT, 0, 0)
-/* 0x6A */ DEFINE_SCENE(sutaru_scene, none, SCENE_SUTARU, SDC_DEFAULT, 0, 0)
-/* 0x6B */ DEFINE_SCENE(hairal_niwa2_scene, g_pn_12, SCENE_HAIRAL_NIWA2, SDC_CASTLE_COURTYARD_GUARDS, 0, 0)
-/* 0x6C */ DEFINE_SCENE(sasatest_scene, none, SCENE_SASATEST, SDC_DEFAULT, 0, 0)
-/* 0x6D */ DEFINE_SCENE(testroom_scene, none, SCENE_TESTROOM, SDC_DEFAULT, 0, 0)
+/* 0x65 */ DEFINE_DEBUG_SCENE(test01_scene, none, SCENE_TEST01, SDC_CALM_WATER, 0, 0)
+/* 0x66 */ DEFINE_DEBUG_SCENE(besitu_scene, none, SCENE_BESITU, SDC_BESITU, 0, 0)
+/* 0x67 */ DEFINE_DEBUG_SCENE(depth_test_scene, none, SCENE_DEPTH_TEST, SDC_DEFAULT, 0, 0)
+/* 0x68 */ DEFINE_DEBUG_SCENE(syotes_scene, none, SCENE_SYOTES, SDC_DEFAULT, 0, 0)
+/* 0x69 */ DEFINE_DEBUG_SCENE(syotes2_scene, none, SCENE_SYOTES2, SDC_DEFAULT, 0, 0)
+/* 0x6A */ DEFINE_DEBUG_SCENE(sutaru_scene, none, SCENE_SUTARU, SDC_DEFAULT, 0, 0)
+/* 0x6B */ DEFINE_DEBUG_SCENE(hairal_niwa2_scene, g_pn_12, SCENE_HAIRAL_NIWA2, SDC_CASTLE_COURTYARD_GUARDS, 0, 0)
+/* 0x6C */ DEFINE_DEBUG_SCENE(sasatest_scene, none, SCENE_SASATEST, SDC_DEFAULT, 0, 0)
+/* 0x6D */ DEFINE_DEBUG_SCENE(testroom_scene, none, SCENE_TESTROOM, SDC_DEFAULT, 0, 0)
```

Comment:
We still need to define the enum values in z64scene.h somehow, because these scenes are still referenced by the entrance table. I think there's probably a better way though, do you have any ideas?

Reply:
I did not know the retail entrance table still had the debug entries bruh

we already have the entrance table referencing a deleted scene, `SCENE_UNUSED_6E`
it's a define, maybe we also do that idk
```c
#if !OOT_DEBUG
#define SCENE_TEST01 110
#define SCENE_BESITU 111
#define SCENE_DEPTH_TEST 112
...
#endif
```

Reply:
Hmm I just realized that SCENE_MAX is wrong currently, so that's another point in favor of just wrapping them in `#if OOT_DEBUG`, it would set SCENE_MAX correctly.

If we go this route I'd like to define `SCENE_UNUSED_6E = 0x6E` for consistency. Right now `SCENE_UNUSED_6E` has this comment:
```
// this define exists to preserve shiftability for an unused scene that is
// listed in the entrance table
#define SCENE_UNUSED_6E SCENE_ID_MAX
```

How important is preserving shiftability (both for SCENE_UNUSED_6E and the debug-only scenes)? Presumably those entrance tables are unused so nobody cares if they point to a random scene, right?

Reply:
I found this thread via git blame: https://github.com/zeldaret/oot/pull/1131#discussion_r799874016

I went ahead and made this change, but maybe @fig02 and/or @mzxrules have ideas

Reply:
I'm okay with the defines
here is a shifting alternative I'd also be okay with if others would like it better:
(untested)
```c
// Fake enum values for scenes that are still referenced in the entrance table
enum {
    SCENE_ID_LAST = SCENE_ID_MAX - 1,
#if !OOT_DEBUG
    // Debug-only scenes
    SCENE_TEST01,
    SCENE_BESITU,
    SCENE_DEPTH_TEST,
    SCENE_SYOTES,
    SCENE_SYOTES2,
    SCENE_SUTARU,
    SCENE_HAIRAL_NIWA2,
    SCENE_SASATEST,
    SCENE_TESTROOM,
#endif
    // Deleted scene
    SCENE_UNUSED_6E
};
```

Reply:
I dont think I really have an opinion here and am fine with whatever you guys think is best. The existing solution for scene 6E was the best compromise at the time for that scenario, im not really attached to it

Reply:
Alright, I think we should go with this for now, and keep Dragorn's suggestion in our back pocket if we want to change it later